onnx/onnxifi.h

```diff
@@ -776,6 +794,7 @@ typedef ONNXIFI_CHECK_RESULT onnxStatus
 typedef ONNXIFI_CHECK_RESULT onnxStatus
   (ONNXIFI_ABI* onnxInitGraphFunction)(
     onnxBackend backend,
+    const uint64_t* auxPropertiesList,
```

Comment:
How do we determine the length of auxPropertiesList?

Reply:
It must be terminated by `ONNXIFI_BACKEND_PROPERTY_NONE`/`ONNXIFI_GRAPH_PROPERTY_NONE`